clang/lib/Driver/Driver.cpp

```diff
@@ -4654,9 +4673,10 @@ InputInfo Driver::BuildJobsForActionNoCache(
     Result = InputInfo(A, BaseInput);
   else {
     std::string OffloadingPrefix;
-    // When generating binaries with -fsycl-link-target, the output file prefix
-    // is the triple arch only
-    if (Args.getLastArg(options::OPT_fsycl_link_targets_EQ)) {
+    // When generating binaries with -fsycl-link-target or -fsycl-link, the
```

Comment:
Do we need specify some triples when we use `fsycl-link`? 
If not, which one will be used?

Reply:
-fsycl-link by itself does not change the compilation flow.  It requires -fsycl which implies spir64-unknown-linux-sycldevice.